src/builders.ts

```diff
@@ -377,12 +380,27 @@ export class BomBuilder {
     return component
   }
 
+  URL_QUALIFIERS_TO_DROP = ['download_url', 'vcs_url']
+
+  private filterQualifiers (qualifiers: { [key: string]: string }): {} | undefined {
```

Comment:
would love to see the type of `qualifiers` be `PackageURL['qualifiers']`
this would make the typing more robust to the foreign type definition.

example: 
```javascript
  private filterQualifiers (qualifiers: Exclude<PackageURL['qualifiers'], undefined | null> = {}): {} | undefined {}
```

Reply:
Good point! This is what happens having been away from javascript and types for most of the last decade 😆